src/test/unit/views/respondent-application-details.test.ts

```diff
@@ -14,17 +14,39 @@ const respondentApplicationDetailsJSONRaw = fs.readFileSync(
 
 const respondentApplicationDetailsJSON = JSON.parse(respondentApplicationDetailsJSONRaw);
 const titleClass = 'govuk-heading-xl govuk-!-margin-bottom-4 govuk-!-margin-top-1';
-const summaryListClass = 'govuk-summary-list__value';
-const summaryListHeadingClass = 'govuk-summary-list__key';
+const summaryListValueClass = 'govuk-summary-list__value';
+const summaryListKeyClass = 'govuk-summary-list__key';
+const summaryListTitleClass = 'govuk-summary-list__key govuk-heading-m govuk-!-margin-top-1';
+//const buttonClass = 'govuk-button';
```

Comment:
commented code should be removed if not in use?

Reply:
Removed comment